src/App.jsx

```diff
@@ -6,11 +6,13 @@ import mermaidMuiThemeConfig from './styles/mermaidMuiThemeConfig.js'
 import MermaidDash from './components/MermaidDash/MermaidDash.jsx'
 import { Auth0Provider } from '@auth0/auth0-react'
 import { FilterProjectsProvider } from './context/FilterProjectsContext'
+import { useState } from 'react'
 
 const App = () => {
   const navigateTo = useNavigate()
   const theme = createTheme(mermaidMuiThemeConfig)
   const location = useLocation()
+  const [isApiDataDoneLoading, setIsApiDataDoneLoading] = useState(false)
```

Comment:
I moved this to global app state to avoid hitting prod with hot reloading when we change the filter context file.